python/requirements/ml/requirements_dl.txt

```diff
@@ -1,7 +1,8 @@
 # These requirements are used for the CI and CPU-only Docker images so we install CPU only versions of torch.
 # For GPU Docker images, you should install requirements_ml_docker.txt afterwards.
 
-tensorflow==2.11.0
+tensorflow==2.11.0; sys_platform != 'darwin' or platform_machine != 'arm64'
+tensorflow-macos==2.11.0; sys_platform == 'darwin' and platform_machine == 'arm64'
```

Comment:
on recent macs you'll probably also want gpu acceleration using `tensorflow-metal` ref https://developer.apple.com/metal/tensorflow-plugin/